### DISCUSSION TITLE
Please delete My account!

### DISCUSSION THREAD

**Discussion by FoodChangesLives** [community/community#21030] [Actions]
I do everything right but your automated process don't work! Quit using my time! Delete it! 

![40E0816A-B418-42B5-B06F-EFFC28EF3BF7](https://user-images.githubusercontent.com/109756918/180278393-66c3869a-bfba-4b58-af3c-77aaf9bea290.png)

**Comment by natashapetrus** [comment]
You should type "delete my account". Lose the "below".

**Comment by FoodChangesLives** [comment]
I tried that! I tried small letters! Tries big letters, tried with spaces!
Without spaces! Tried it all!

On Thu, Jul 21, 2022 at 10:42 Natasha Theodora Petrus <
***@***.***> wrote:

> You should type "delete my account". Lose the "below".
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/github-community/discussions/21030#discussioncomment-3200127>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2FMD5TJT2HDAKJOWFHPLATVVGDYHANCNFSM54IOV7ZQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> .com>
>
-- 

Michaela Morrell
Swedish Nutritionist, CEO Morrell AS, Food Changes Lives
(602)524-9388
| www.FoodChangesLives.com  <http://www.FoodChangesLives.com>
***@***.***  ***@***.***>
20740 West Founder Circle, 85396 Buckeye
<https://maps.google.com/?q=20740+West+Founder+Circle,+85396+Buckeye>
https://calendly.com/michaela_morrell/let-s-connect
<https://calendly.com/michaela_morrell/let-s-connect>
<https://www.facebook.com/michaela.morrell>
<https://www.instagram.com/michaela_morrell/>
<http://www.linkedin.com/in/michaela-morrell>
<https://www.youtube.com/c/FoodChangesLives>
<https://webapp.wisestamp.com/editor>

Create your own email signature
<https://www.wisestamp.com/create-own-email-signature/?utm_source=promotion&utm_medium=signature&utm_campaign=create_your_own&srcid=>

**Comment by adiansyahputra** [comment]
check this https://howtodelete.org/delete-github-account/

**Reply by adiansyahputra** [reply]
i hope that work for you

**Comment by MrSarno** [comment]
It should also prompt you to confirm your password.

Is it possible for you to scroll down in the pop-up window? If it isn't, try using a different web browser in case an extension is interfering.

**Reply by MrSarno** [reply]
Here's a screenshot;

<img width="451" alt="Screenshot 2022-07-22 at 17 04 02" src="https://user-images.githubusercontent.com/32073555/180479119-e235f12f-6794-40a1-a070-dcea55017bd5.png">
